src/dmd/root/array.d

```diff
@@ -253,3 +253,63 @@ private:
     size_t len;
     size_t *ptr;
 }
+
+/**
+ * Converts the given root Array to the standard D array.
+ * Params:
+ *  array = the array to convert.
+ * Returns:
+ *  The given array converted to a standard D array.
+ */
+@property T[] toDArray(T)(ref Array!T array)
+{
+    return array.data[0..array.dim];
+}
+
+/**
+ * Splits the array at $(D index) and expands it to make room for $(D length)
+ * elements by shifting everything past $(D index) to the right.
+ * Params:
+ *  array = the array to split.
+ *  index = the index to split the array from.
+ *  length = the number of elements to make room for starting at $(D index).
+ */
+void split(T)(ref Array!T array, size_t index, size_t length)
```

Comment:
This seems to only be here to support the unittest, but it is a public symbol.  Please embed it in the unittest or encase it in `version(unittest)`.

Reply:
I assume this comment was meant for  the `toDArray` function?

Reply:
Yes, sorry!